### PR TITLE
docs(css): Add details about subgrid to Grid Learn article

### PR DESCRIPTION
**Pull request by bsmth**
We don't mention subgrid or nesting grids in the learn doc, this PR adds some more examples and links to subgrid where possible.

__additions:__
- nested grid example 
- example using `subgrid`
- link to subgrid in See Also
 
__changes:__
- Height of embedded examples to make vertical space a little tighter
- Some flyby tidy up

__follow-ups:__
It might be nice to add the example with nested grids to be available to download in the learning area. Pending review changes on that.

### CODE REVIEW

File: files/en-us/learn/css/css_layout/grids/index.md [hunk 1]
```diff
@@ -587,9 +584,194 @@ The rules for `grid-template-areas` are as follows:
...
+}
+```
+
+{{ EmbedLiveSample('subgrid', '100%', 300) }}
+
 ## Grid frameworks in CSS Grid
 
-Grid "frameworks" tend to be based around 12 or 16-column grids. With CSS Grid, you don't need any third party tool to give you such a framework — it's already there in the spec.
+There are many grid frameworks available that provide a 12 or 16-column grid to help lay out your content.
+The good news is that you probably don't need any third-party frameworks to help you build grid-based layouts — it's already in the spec and supported in most modern browsers.
```
Comment (dipikabh) [inline files/en-us/learn/css/css_layout/grids/index.md:774]:
```suggestion
Numerous grid frameworks are available, offering a 12 or 16-column grid, to help with laying out your content.
The good news is that you probably won't need any third-party frameworks to help you create grid-based layouts — grid functionality is already included in the specification and is supported by most modern browsers.
```

File: files/en-us/learn/css/css_layout/grids/index.md [hunk 2]
```diff
@@ -700,6 +882,7 @@ In this overview, we've toured the main features of CSS Grid Layout. You should
 ## See also
 
 - [CSS Grid guides](/en-US/docs/Web/CSS/CSS_grid_layout#guides)
+- [Subgrid](/en-US/docs/Web/CSS/CSS_grid_layout/Subgrid)
 - [CSS Grid Inspector: Examine grid layouts](https://firefox-source-docs.mozilla.org/devtools-user/page_inspector/how_to/examine_grid_layouts/index.html)
 - [CSS-Tricks Guide to Grid](https://css-tricks.com/snippets/css/complete-guide-grid/) — an article explaining everything about Grid in a visually appealing way
 - [Grid Garden](https://cssgridgarden.com/) — an educational game to learn and better understand the basics of Grid
```
Comment (dipikabh) [inline files/en-us/learn/css/css_layout/grids/index.md:888]:
```suggestion
- A [list of guides](/en-US/docs/Web/CSS/CSS_grid_layout#guides) related to the CSS grid layout
- [Subgrid](/en-US/docs/Web/CSS/CSS_grid_layout/Subgrid) guide
- [CSS grid inspector: Examine grid layouts](https://firefox-source-docs.mozilla.org/devtools-user/page_inspector/how_to/examine_grid_layouts/index.html) on firefox-source-docs
- [A complete guide to CSS grid](https://css-tricks.com/snippets/css/complete-guide-grid/), a visual guide on CSS-Tricks (2023)
- [Grid Garden](https://cssgridgarden.com/), an educational game to learn and better understand the basics of grid  on cssgridgarden.com
```

**Review by dipikabh — CHANGES_REQUESTED** [review]
Thanks @bsmth for these updates and additions!

**Comment by bsmth** [comment]
Thank you for the suggestions 🎉

**Review by dipikabh — APPROVED** [review]
Thanks for the updates 👍

**Pull request merged**